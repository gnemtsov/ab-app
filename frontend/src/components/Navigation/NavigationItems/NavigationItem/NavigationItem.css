.NavigationItem {
    margin: 0.7rem;
    display: block;
    width: 100%;
}

.NavigationItem a {
    color: #2534e4;
    text-decoration: none;
    width: 100%;
    display: block;
}

.NavigationItem a:hover,
.NavigationItem a:active,
.NavigationItem a.active {
    color: #FF9800;
}

@media (min-width: 500px) {
    .NavigationItem {
        margin: 0.7rem;
        display: flex;
        height: 100%;
        width: auto;
        align-items: center;
    }
    
    .NavigationItem a {
        color: white;
        height: 100%;
        padding: 0.2rem;
        border-bottom: 2px solid transparent;
    }
    
    .NavigationItem a:hover,
    .NavigationItem a:active,
    .NavigationItem a.active {
        border-bottom: 2px solid #FF9800;
    }
}