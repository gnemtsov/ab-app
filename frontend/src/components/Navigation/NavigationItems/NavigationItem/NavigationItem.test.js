import React from 'react';

import { configure, render, mount } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { BrowserRouter } from "react-router-dom";

import NavigationItem from './NavigationItem';

configure({ adapter: new Adapter() });

describe('NavigationItem', () => {
	it('render with exact', () => {
		const wrapper = render(
			<BrowserRouter>
				<NavigationItem link="/" exact>Departments</NavigationItem>
			</BrowserRouter>
		);
		
		expect(wrapper).toMatchSnapshot();
	});
	
	it('render without exact', () => {
		const wrapper = render(
			<BrowserRouter>
				<NavigationItem link="/">Departments</NavigationItem>
			</BrowserRouter>
		);
		
		expect(wrapper).toMatchSnapshot();
	});
});
