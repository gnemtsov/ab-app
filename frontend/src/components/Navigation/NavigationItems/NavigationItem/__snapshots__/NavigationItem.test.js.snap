// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationItem render with exact 1`] = `
<li
  class="NavigationItem"
>
  <a
    aria-current="page"
    class="active"
    href="/"
  >
    Departments
  </a>
</li>
`;

exports[`NavigationItem render without exact 1`] = `
<li
  class="NavigationItem"
>
  <a
    aria-current="page"
    class="active"
    href="/"
  >
    Departments
  </a>
</li>
`;
